src/metabase/models/setting.clj

```diff
@@ -1371,8 +1371,16 @@
                 ;; err on the side of caution
                 true))
 
-    ;; TODO: handle the remaining formats explicitly
-    true))
+    :boolean false
+    :csv (not (instance? java.io.EOFException ex))
+    :double true
+    :keyword true
+    :integer true
+    :positive-integer true
+    :timestamp true
+
+    (do (log/warn "Redaction criteria not defined for type and obtain an exception of type" (:type setting) (type ex))
```

Comment:
I think we can skip this log. logs should make troubleshooting easier, not for leaving TODOs for us developers

Reply:
My desire was for it to prompt anyone writing a new settings format to "dot the i", but thinking about it more I see how it could easily not be triggered on the dev branch, fall through into a release, and be seen by a user.

Do you think it's fine to remove this without adding any other "coverage protection" mechanism?